elide-core/src/main/java/com/yahoo/elide/core/filter/expression/FilterExpressionVisitor.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2017, Oath Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.core.filter.expression;
+
+/**
+ * Visitor for filterExpression .
+ * @param <T> The return type of the visitor
+ */
+public interface FilterExpressionVisitor<T> extends Visitor<T> {
```

Comment:
You shouldn't need to define this.  You can call FilterExpression.accept on the visitor you created below to start the visitation.